Dockerfile.openpilot_base

```diff
@@ -12,19 +12,19 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
-ENV PIPENV_SYSTEM=1
+ENV POETRY_SYSTEM=1
 ENV PYENV_VERSION=3.8.10
 ENV PYENV_ROOT="/root/.pyenv"
 ENV PATH="$PYENV_ROOT/bin:$PYENV_ROOT/shims:$PATH"
 
-COPY Pipfile Pipfile.lock .python-version update_requirements.sh /tmp/
+COPY pyproject.toml poetry.lock .python-version update_requirements.sh /tmp/
```

Comment:
is the cache still being remove properly here?

Reply:
I think the poetry cache is in `$HOME/.cache` so should be deleted